src/components/reusable/inlineCodeView/inlineCodeView.ts

```diff
@@ -0,0 +1,42 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import InlineCodeViewStyles from './inlineCodeView.scss';
+
+/**
+ * `<kyn-inline-code-view>` component to display code snippets inline within HTML content.
+ * @slot unnamed - inline code snippet slot.
+ */ @customElement('kyn-inline-code-view')
+export class InlineCodeView extends LitElement {
+  static override styles = InlineCodeViewStyles;
+
+  /** Sets background and text theming. */
+  @property({ type: String })
+  darkTheme: 'light' | 'dark' | 'darker' = 'darker';
```

Comment:
Did Robert design two dark variants?

Reply:
robert liked the way the original hex value for the background looked in figma, but said that the contrast wasn't quite what he wanted when implemented when viewed in storybook, so he provided that slightly darker hex value.

my OneDark pro theme has light, dark, darker theming, so i figured i would add that middle one as an option. i can remove. 

we got into a bit of a ux purgatory here, initially agreeing that ux wasn't required apart from the syntax colors, so i might be taking some design liberties on this one more than would otherwise, apologies